src/data/services/LmsApiService.js

```diff
@@ -0,0 +1,18 @@
+import apiClient from '../apiClient';
+
+class LmsApiService {
+  static baseUrl = process.env.LMS_BASE_URL;
+
+  static fetchProgramEnrollmentOverview(programUUID) {
+    const programEnrollmentOverviewUrl = `${LmsApiService.baseUrl}/api/program_enrollments/v1/programs/${programUUID}/overview`;
+    // TODO: Uncomment line below and remove empty promise.
```

Comment:
@adamstankiewicz @irfanuddinahmad I am getting redirect to LMS for login even though I am logged in. I have added todo once redirect issue is fixed and authentication works.